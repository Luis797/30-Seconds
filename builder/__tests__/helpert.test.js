const {sortByLevel} = require("../helpers");

test('sortByLevel :: sorts according to schema ', () => {
  const initial = [
    {level: 'intermediate',},
    {level: 'beginner',},
    {level: 'advanced',},
    {level: 'intermediate',},
  ];
  const sorted = sortByLevel(initial);

  expect(sorted).toMatchSnapshot();

  // Does not mutate the original array
  expect(initial).toMatchSnapshot();
});
