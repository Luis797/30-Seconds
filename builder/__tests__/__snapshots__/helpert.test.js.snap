// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sortByLevel :: sorts according to schema  1`] = `
Array [
  Object {
    "level": "beginner",
  },
  Object {
    "level": "intermediate",
  },
  Object {
    "level": "intermediate",
  },
  Object {
    "level": "advanced",
  },
]
`;

exports[`sortByLevel :: sorts according to schema  2`] = `
Array [
  Object {
    "level": "intermediate",
  },
  Object {
    "level": "beginner",
  },
  Object {
    "level": "advanced",
  },
  Object {
    "level": "intermediate",
  },
]
`;
